src/tests/nativeaot/DesktopStackSize/DesktopStackSize.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We should not need this test regular for native AOT. The other test that you are adding is part of regular native AOT test runs.